packages/core/src/models/LibrariesState.ts

```diff
@@ -0,0 +1,46 @@
+import * as t from 'io-ts';
```

Comment:
if this file represent the LibraryState but we don't have a type LibraryState being exported, this is a bit weird I think.

Reply:
I don't think so, if you have a module `utils` you don't have to export a const or a function named `utils`.
It's just what it represents but it contains way more than one type, otherwise I would have put a default export

Reply:
but then it becomes uneasy to understand.
Honestly I wouldn't know what's a LibrariesStates and what's on the file withouth the function in SM-UI that writes in the file.

So if this is to be used outside then I find it totally un-understandable and wrong.